lib/tasks/licenses.rake

```diff
@@ -37,14 +55,8 @@ namespace :licenses do  # rubocop:disable Metrics/BlockLength
 
   private
 
-  def cli
-    @cli ||= begin
-               require 'license_finder'
-               LicenseFinder::CLI::Main.new
-             end
-  end
```

Comment:
Why remove this and create a new instance of `LicenseFinder::CLI::Main.new` every time?

Reply:
It won't write the temporary report if I don't create a new instance. Anyway I'm shooting in the dark, trying to make it work, expect weird stuff 💩

Reply:
Is it not reviewable yet?

Reply:
Very reviewable